packages/core/frontend/sirius-components-core/src/diagram/DiagramExtensionPoints.ts

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo and others.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { ReactFlowProps } from 'reactflow';
+import { DataExtensionPoint } from '../extension/ExtensionRegistry.types';
+
+export const diagramRendererReactFlowPropsCustomizerExtensionPoint: DataExtensionPoint<
+  Array<ReactFlowPropsConfigurer>
+> = {
+  identifier: 'diagram#reactFlowContribution',
+  fallback: [],
+};
+
+export type ReactFlowPropsConfigurer = (options: ReactFlowProps) => ReactFlowProps;
```

Comment:
We might want to also send the current nodes/edges and the payload ?

Reply:
Have a look at my examples, I can already access the nodes and edges, I'm just missing the payload especially if I want to customize the subscription but this contribution already brings interesting new capabilities.